TeamCharles/workforce-management
# 58-editComputerForm - Edit a Computer

## Status
**READY**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

## Resolves Issue Number

https://github.com/TeamCharles/workforce-management/issues/58

## Deploy Notes

```sh
git pull --prune
git checkout 58-editComputerForm
dotnet run
```

## Steps to Test or Reproduce

1. Navigate to `localhost:5000`

1. Click the "Computers" button in the navigation

1. Click the "Edit" button on a Computer

1. Attempt to edit a computer

1. Edit the computer successfully

1. See the updated info in `localhost:5000/Computer`

## Impacted Files in Application

* Views/Computer/Index.cshtml

* Views/Computer/Edit.cshtml

* ViewModels/Computer/ComputerEdit.cs

* Controllers/ComputerController.cs
